docs/src/examples/components/Dropdown/Types/DropdownExampleSingleSelection.shorthand.steps.ts

```diff
@@ -1,13 +1,19 @@
-import { Dropdown, Button, List } from '../../../../../../packages/react/src'
+import { Dropdown } from '../../../../../../packages/react/src'
```

Comment:
#837 was merged, now we can use package import:

```suggestion
import { Dropdown } from '@stardust-ui/react'
```

Reply:
done